reference/gdscript.rst

```diff
@@ -6,14 +6,15 @@ GDScript
 Introduction
 ------------
 
-GDScript is a high level, dynamically typed programming language used to
-create content. It uses a syntax that is very similar to the Python
-language (blocks are indent-based) and its goal is to be very optimal
-and tightly integrated with the engine, allowing great flexibility for
-content creation and integration.
+*GDScript* is a high level, dynamically typed programming language used to
+create content. It uses a syntax similar to 
+`Python <https://en.wikipedia.org/wiki/Python_%28programming_language%29>`_ 
+(blocks are indent-based and many keywords are similar). Its goal is 
+to be optimized for and tightly integrated with the Godot engine, allowing 
+great flexibility for content creation and integration.
 
 History
--------
+^^^^^^^
```

Comment:
Ah, you are right, the Example is too far in. I'll move it out one level. I don't think the previous structure made too much sense either though - everything was top-level making it quite hard to see what belonged to where. The structure I was aiming for is this: 

1. GDScript
  1. Introduction
     1. History
     1. Example of GDScript
  1. Language